mimic/rest/loadbalancer_api.py

```diff
@@ -54,10 +55,104 @@ def resource_for_region(self, region, uri_prefix, session_store):
         Get an :obj:`twisted.web.iweb.IResource` for the given URI prefix;
         implement :obj:`IAPIMock`.
         """
+        self.session_store = session_store
```

Comment:
Removing this logic causes exceptions to be thrown elsewhere.  If this is a significant concern, I recommend opening a technical debt ticket as a separate action item.